app/src/containers/MovieDetailsPage/MovieListTitle/MovieListTitle.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './MovieListTitle.scss';
+
+export default class MovieListTitle extends React.Component {
```

Comment:
Can be a stateless functional component.

Reply:
done